recipe/meta.yaml

```
@@ -32,6 +31,17 @@ requirements:
     - pytest-runner
     - python
     - pytorch >=1.7
```

I think PyTorch is globally pinned as of 1.9.0. Is that right @hmaarrfk?

```suggestion
    - pytorch
```

---

pytorch as a build time dependency should be left to conda-forge-pinnings.

This suggestion is the way to go (I'm a maintainer in the pytorch recipe).